### ISSUE TITLE
Red highlights remains long after appropriate

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap#2788]
With the stepper turned on my command stopped (expected a list instead of a number). So I turned off the stepper and fixed the bug. But the blocks were still highlighted in red. As you can see below even after editing the command the highlights remain (but are no longer at the right place).

![image](https://user-images.githubusercontent.com/2138696/109141974-44a07480-7799-11eb-8533-c2c4c78f0da6.png)

I was using my copy of Snap! 6.5.0

**Comment by jmoenig** [comment]
Thanks, Ken. These red halos are tricky, because they stay around even after the process has been terminated. usually pressing the stop button removes them for good. Did you try that?

**Comment by ToonTalk** [comment]
I'm pretty sure I did try the stop button. Probably while the editor was still showing it.

**Comment by ToonTalk** [comment]
At least editing it should remove the highlights since they don't even track the new script.

**Comment by jmoenig** [comment]
yeah, probably. Tricky part is that we don't really want to check for the halo at every keystroke all the time. But yes, red halos should not stay around for too long.
